cve_bin_tool/output_engine/__init__.py

```diff
@@ -89,7 +89,7 @@ def output_file(self, output_type="console"):
 
         # Check if we need to generate a filename
         if not self.filename:
-            self.filename = generate_filename(self.filename, output_type)
+            self.filename = generate_filename(output_type)
```

Comment:
Thanks for catching that. But I fixed that but I don't know how it showed up here maybe I forgot to save the file or something.